core/client/src/main/java/alluxio/client/file/options/DeleteOptions.java

```diff
@@ -48,8 +51,14 @@ public boolean isRecursive() {
   }
 
   /**
-   * Sets the recursive flag.
-   *
+   * @return the mAlluxioOnly flag value; the flag specifies whether the file content should
```

Comment:
nit: I would prefer to avoid using variable names in javadoc ... the javadoc could say:

```
@return the value of the flag that indicates whether the file should be deleted in Alluxio only, or in UFS as well
```